sonic-xcvrd/xcvrd/xcvrd_utilities/media_settings_parser.py

```diff
@@ -17,6 +18,9 @@
 VENDOR_KEY = 'vendor_key'
 MEDIA_KEY = 'media_key'
 LANE_SPEED_KEY = 'lane_speed_key'
+DEFAULT_KEY = 'Default'
+# This is useful if default value is desired when no match is found for lane speed key
```

Comment:
@longhuan-cisco not able to understand. Can you please explain with an example. Can you capture these in the HLD:- https://github.com/sonic-net/SONiC/blob/master/doc/media-settings/Media-based-Port-settings.md